# HG changeset patch
# User Rik <rik@octave.org>
# Date 1344040685 25200
#      Fri Aug 03 17:38:05 2012 -0700
# Node ID 2512448babac252dfbe0fb31960ef6de040e2315
# Parent  8523df595d42d46603aabd1bdb7371e16c29bebe
image.m: Remove input validation tests which cause window to pop up.

* image.m: Remove input validation tests which cause window to pop up.

diff --git a/scripts/image/image.m b/scripts/image/image.m
--- a/scripts/image/image.m
+++ b/scripts/image/image.m
@@ -241,11 +241,8 @@ endfunction
 %! imagesc (g, g, sin (h));
 %! plot (g, 11.0 * ones (size (g)));
 %! imagesc (g, g+12, cos (h/2));
 %! plot (g, 11.5 * ones (size (g)));
 %! axis ([0 10 0 22]);
 %! hold off;
 %! title ("line, image, line, image, line");
 
-## Test input validation
-%!error <can not be complex> image (1+i)
-%!error <matrix is empty> image ([])
